Python/Python.sublime-syntax

```diff
@@ -15,38 +16,205 @@ file_extensions:
+            - match: ''
+              set:
+                - meta_scope: meta.statement.import.python
+                - include: line_continuation_or_pop
+                - include: import_name_list
+        - include: dotted_name
+
+  import_name_list:
+    - match: ','
+      scope: punctuation.separator.import-list.python
+    - include: import_alias
+    - include: name
+    - match: \*
+      scope: keyword.other.import-all.python
+    - match: \S+
+      scope: invalid.illegal.name.import.python
+
+  import_alias:
+    - match: \b(as)\b
+      scope: keyword.control.import.as.python
+
+  block_statements:
+    # async for ... in ...:
+    - match: \b(async +)?(for)\b
+      captures:
+        1: keyword.control.flow.async.python
+        2: keyword.control.flow.for.python
+      push:
+        - meta_scope: meta.statement.for.python
+        - include: line_continuation_or_pop
+        - match: \b(in)\b
+          scope: keyword.control.flow.for.in.python
+          set:
+            - meta_scope: meta.statement.for.python
+            - include: line_continuation_or_pop
+            - match: ':'
+              scope: punctuation.definition.block.for.python
+              pop: true
+            - include: expressions
+        - match: ':'
+          scope: invalid.illegal.missing-in.python
+          pop: true
+        - include: target_list
+    # async with ... as ...:
+    - match: \b(async +)?(with)\b
+      captures:
+        1: keyword.control.flow.async.python
+        2: keyword.control.flow.with.python
+      push:
+        - meta_scope: meta.statement.with.python
+        - include: line_continuation_or_pop
+        - match: \b(as)\b
+          scope: keyword.control.flow.with.as.python
+          set:
+            - meta_scope: meta.statement.with.python
+            - include: line_continuation_or_pop
+            - match: ':'
+              scope: punctuation.definition.block.with.python
+              pop: true
+            - include: name
+        - match: ':'
+          scope: punctuation.definition.block.with.python
+          pop: true
+        - include: expressions
+    # except ... as ...:
+    - match: \b(except)\b
+      scope: keyword.control.flow.except.python
+      push:
+        - meta_scope: meta.statement.except.python
+        - include: line_continuation_or_pop
+        - match: ':'
+          scope: punctuation.definition.block.except.python
+          pop: true
+        - match: '\b(as)\b'
+          scope: keyword.control.flow.as.python
+          set:
+            - meta_scope: meta.statement.except.python
+            - include: line_continuation_or_pop
+            - match: ':'
+              scope: punctuation.definition.block.except.python
+              pop: true
+            - include: name
+        - include: target_list
+    - match: \b(elif|else|finally|if|try|while)\b
+      scope: keyword.control.flow.python
+
   expressions:
+    # Always include this last!
```

Comment:
To `expressions`. The comment doesn't make sense if it was for `comment`, which is the first include.